js/foundation.tabs.js

```diff
@@ -114,9 +93,32 @@ class Tabs {
       }
     }
 
+    this._checkDeepLink = this._checkDeepLink.bind(this);
```

Comment:
So that we can 
1. Count on `this` within the function being a reference to the plugin
2. have a single representation of the function (rather than doing bind at the time we add the handler), letting us remove it using `off`